src/NosCore.GameObject/Networking/ClientSession/ClientSession.cs

```diff
@@ -120,18 +122,18 @@ public class ClientSession : NetworkClient, IClientSession
 
         public AccountDto Account { get; set; }
 
-        public Character? Character
+        public Character Character
         {
             get
             {
                 if ((_character != null) && !HasSelectedCharacter)
                 {
                     // cant access an
                     _logger.Warning(LogLanguage.Instance.GetMessageFromKey(LogLanguageKey.CHARACTER_NOT_INIT));
-                    return null;
+                    throw new NullReferenceException();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the exception throwing from this property getter, or refactor the property into a method.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)